src/main/java/com/hillel/items_exchange/security/jwt/JwtTokenProvider.java

```diff
@@ -60,20 +63,25 @@ private String getUsername(String token) {
 
     public String resolveToken(HttpServletRequest req) {
         String bearerToken = req.getHeader(AUTHORIZATION_HEADER_NAME);
-        if (bearerToken != null && bearerToken.startsWith(BEARER_PREFIX)) {
-            return bearerToken.substring(7);
+        if(bearerToken != null) {
+            if(bearerToken.startsWith(BEARER_PREFIX)) {
+                return bearerToken.substring(7);
```

Comment:
Looks like 'magic number' you may use here constant or BEARER_PREFIX.length.

Reply:
Fixed